src/styles/blocks/star.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
```

Comment:
The use of `nth-child(-n + 4)` is correct for selecting the first four children. Ensure that the parent element `.stars--rating-4` has at least four `.stars__star` children, or this selector might not work as intended.